server/internal/usecase/interactor/integration_test.go

```diff
@@ -0,0 +1,583 @@
+package interactor
+
+import (
+	"context"
+	"net/url"
+	"regexp"
+	"testing"
+	"time"
+
+	"github.com/reearth/reearth-cms/server/internal/infrastructure/memory"
+	"github.com/reearth/reearth-cms/server/internal/usecase"
+	"github.com/reearth/reearth-cms/server/internal/usecase/interfaces"
+	"github.com/reearth/reearth-cms/server/pkg/id"
+	"github.com/reearth/reearth-cms/server/pkg/integration"
+	"github.com/reearth/reearth-cms/server/pkg/user"
+	"github.com/reearth/reearthx/rerror"
+	"github.com/samber/lo"
+	"github.com/stretchr/testify/assert"
+)
+
+func testSuite() (now time.Time, op *usecase.Operator, uri *url.URL, uId id.UserID, iId1, iId2 id.IntegrationID, i1, i2 *integration.Integration) {
```

Comment:
The number of return values is too large and should be summarized by defining struct.